xenvm.xml

```diff
@@ -6,6 +6,7 @@
   <remote  name="epam"
            fetch="ssh://git@gitpct.epam.com/epmd-aepr/" />
   <remote name="fd" fetch="https://gitlab.freedesktop.org/" />
+  <remote name="gitbud_epam_com" fetch="ssh://git@gitbud.epam.com"/>
```

Comment:
Hi @arminn, please squash this commit with this one: https://github.com/xen-troops/android_manifest/pull/86/commits/87d9097ed6d151cc45df5c19cdfbf7c4ba704cc6